wappsto/connection/seluxit_rpc.py

```diff
@@ -761,63 +711,44 @@ def create_json_message(
             get: Defines if the request is of type GET. (default: {False})
 
         """
-        network_parameters = {'url': '', 'data': data}
-        base_url = '/network/{}/device/{}/value/'.format(
-            network_id,
-            device_id
-        )
+        base_url = '/network/{}/device/{}/value/'.format(network_id,device_id)
         if put:
             if get:
                 if state == 'state':
-                    network_parameters['url'] = "{}{}/{}/{}".format(
-                        base_url,
-                        value_id,
-                        state,
-                        state_id
-                    )
+                    url = "{}{}/{}/{}".format(base_url,value_id,state,state_id)
                 else:
-                    network_parameters['url'] = "{}{}".format(
-                        base_url,
-                        value_id
-                    )
+                    url = "{}{}".format(base_url,value_id)
 
-                self.data_json_rpc['method'] = 'GET'
                 if trace_id:
-                    network_parameters['url'] = "{}?trace={}".format(
-                        network_parameters['url'],
-                        trace_id
-                    )
+                    self.data_json_rpc = Request('GET',
+                                             url = "{}?trace={}".format(url,trace_id),
+                                             data = data)
+                else:
+                    self.data_json_rpc = Request('GET',
+                                             url = url,
+                                             data = data)
             else:
                 if state == 'state':
-                    network_parameters['url'] = "{}{}/{}/{}".format(
-                        base_url,
-                        value_id,
-                        state,
-                        state_id
-                    )
+                    url = "{}{}/{}/{}".format(base_url,value_id,state,state_id)
                 else:
-                    network_parameters['url'] = "{}{}".format(
-                        base_url,
-                        value_id
-                    )
-                self.data_json_rpc['method'] = 'PUT'
+                    url = "{}{}".format(base_url,value_id)
+                    
                 if trace_id:
-                    network_parameters['url'] = "{}?trace={}".format(
-                        network_parameters['url'],
-                        trace_id
-                    )
+                    self.data_json_rpc = Request('PUT',
```

Comment:
I think all of this, could be rewritten to only use a single "Request" and then just update the parameters for it instead.